pkg/kubelet/eviction/helpers.go

```diff
@@ -513,8 +523,12 @@ func (ms *multiSorter) Less(i, j int) bool {
 
 // priority compares pods by Priority, if priority is enabled.
 func priority(p1, p2 *v1.Pod) int {
-	priority1 := schedulerutils.GetPodPriority(p1)
-	priority2 := schedulerutils.GetPodPriority(p2)
+	if !utilfeature.DefaultFeatureGate.Enabled(features.PodPriority) {
```

Comment:
Did this check come in because of a rebase?

Reply:
Yes. The recent rebase added this definition.

Reply:
Hmm, if this check was added and you did a rebase, I would not expect it to show up as an added line in the diff.

It looks like what actually happened is that this check was removed in https://github.com/kubernetes/kubernetes/pull/79262 but is getting re-added in this PR. Could you remove it?

Reply:
I have removed the feature gate. Please review now.